sdk/translation/Azure.AI.Translation.Document/tests/samples/SampleSnippets.cs

```diff
@@ -59,26 +62,30 @@ public void BadRequestSnippet()
         [Ignore("Samples not working yet")]
         public void DocumentTranslationInput()
         {
+            #region Snippet:DocumentTranslationSingleInput
+#if SNIPPET
+            Uri sourceSasUri = <source SAS URI>;
```

Comment:
Nit: why not use the same format for the snippet as the actual executed code? At least it will compile, and you wouldn't even need the preproc condition.

Reply:
True. I was leaving it as is so when we turn on the samples we will pass real values. But not sure when we will do this, so I changed it based on your suggestions.
Thanks!